src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -43,6 +43,14 @@ const WSTRING agent_port = "DD_TRACE_AGENT_PORT"_W;
 // Sets the "env" tag for every span.
 const WSTRING env = "DD_ENV"_W;
 
+// Sets the "version" tag for every span.
+const WSTRING version = "DD_VERSION"_W;
```

Comment:
Could rename to `service_version` for consistency with the suggestion to rename the TracerSetting.

Reply:
Good idea, done in 28b54ba3c14a8eba2610de65bb218cbac04a56fc